utils/rtl_433/Makefile

```diff
@@ -0,0 +1,45 @@
+# 
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rtl_433
+PKG_VERSION:=6531ea4
+PKG_RELEASE:=$(PKG_SOURCE_VERSION)
```

Comment:
Any reason to not go with numbered releases?

Reply:
Last one was 15 months ago, but there's been plenty of activity in the meantime... Can you ping the maintainer about another release?

Reply:
Yeah that's exactly the reason why. I also mentioned it in my commit message.
I could ping the maintainer see if he has plans for a release. But if not does that have any impact on my pull-request? I hope not, it's a very nice tool.

To be more precise I chose this version because I just opened a pull-request in Domoticz to support new sensors from rtl_433 and therefor I needed a up-to-date version of rtl_433.

Reply:
I noticed this issue topic: https://github.com/merbanan/rtl_433/issues/470
The maintainer left a response quite some time ago, so he is probably aware of the release/versioning issue but has not come up with a solution yet :(